src/Components/LoginInput/Components/LoginDescription.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+class LoginDescription extends Component {
+  render() {
+    return <HyperText herf="">Lost your password?</HyperText>;
+  }
+}
+
+export default LoginDescription;
+
+const HyperText = styled.a`
```

Comment:
```suggestion
const HyperText = styled(Link)`
```
링크 컴포넌트를 상속해서 스타일링 해주세요

Reply:
알겠습니다